modules/dynamic-plugins/proc-install-plugins-rhdh-container-image.adoc

```diff
@@ -0,0 +1,42 @@
+[id="proc-install-plugins-rhdh-container-image_{context}"]
+= Installing plugins included in {product-very-short} container image
+
+In the {product-very-short} container image, a set of dynamic plugins is pre-loaded to enhance functionality. However, due to mandatory configuration requirements, most of the plugins are disabled.
+
+You can activate and configure the plugins in the {product-very-short} container image, including how to manage the default configuration, set necessary environment variables, and ensure the proper functionality of the plugins within your application.
+
+.Prerequisites
+* You have access to the link:https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins.default.yaml[`dynamic-plugins.default.yaml`] file, which lists all preloaded plugins.
+* You have deployed the {product-very-short} application, and have access to the logs of the `install-dynamic-plugins` init container.
+* You have the necessary permissions to modify plugin configurations and access the application environment.
+* You have identified and set the required environment variables referenced by the plugin's default configuration. These environment variables must be defined in the Helm Chart or Operator configuration.
+
+.Procedure
+. Locate the link:https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins.default.yaml[`dynamic-plugins.default.yaml`] file in your setup.
+. Start your {product-very-short} application and access the logs of the `install-dynamic-plugins` init container within the {product-very-short} pod.
+. Look for log entries indicating plugins that are disabled by default. For example:
```

Comment:
I don't think that getting list of the plugins from logs is a user-friendly way to get the list of available plugins. 

We list of the plugins in our documentation (https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/dynamic_plugins_reference/con-preinstalled-dynamic-plugins) so we should probably guide users use this instead parsing logs.